tests/unit/test_config.py

```
@@ -0,0 +1,172 @@
+        assert hasattr(config, "noprint_formats")
+        assert hasattr(config, "noprint")
+        assert config.noprint_formats is True
+        assert config.noprint is False
+
+
+class TestAppConfig:
+    """Tests for AppConfig dataclass."""
+
+    def test_has_correct_attributes(self) -> None:
+        """AppConfig should have the expected attributes."""
+        config = AppConfig(save_data_path="/tmp")
```

Ruff (0.14.14)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)